GSA/openacr-editor
# Output HTML and Markdown

Closes https://github.com/GSA/openacr/issues/119

This is an initial version that recreate the handlebar templates in Svelte for Markdown and HTML. Will have to add to it when we add the chapters and table fields. For Markdown had to build up a string manually to get the correct spacing and line breaks.

QA steps:

1. After deployment ('Deploy' action) is successful, open site https://gsa.github.io/openacr-editor/.
2. Click 'Start new report'.
3. Switch to the 'About' page.
4. Type in values and they can optionally match https://github.com/GSA/openacr/blob/main/openacr/drupal-9.yaml.
4. Click the 'View report' or 'Report' tab.
5. Confirm you see a 'Valid' message.
6. Confirm the HTML on the page displays the various sections except tables, chapters, related OpenACRs. Use https://federalist-02947dd8-86df-467a-b2af-4c5b94c5b1f0.app.cloud.gov/site/gsa/opat-website/openacr/drupal-9.html as an example to compare against.
7. Click the 'Download Report (HTML)' and confirm the HTML file downloaded matches the page.
8. Click the 'Download Report (Markdown)' and confirm the Markdown file downloaded matches the page.
